controllers/users.js

```diff
@@ -64,7 +64,62 @@ class UserController {
 
   async updateUser(request, response) {
     try {
-      const user = await userService.changeUser(request.body);
+      const { userName } = request.body;
+
+      const user = await userService.changeUser(request.body, userName);
+
+      return response.status(200).json({
+        status: 'Success',
+        message: 'User data updated.',
+        user,
+      });
+    } catch (error) {
+      if (error instanceof ValidationError) {
+        return response.status(422).json({
+          status: 'Failure.',
+          message: extractErrors(error),
+        });
+      } else if (error instanceof InexistentItem) {
```

Comment:
The whole point of having custom erros with response code attached to them is to get rid of this verifications. You can catch that validation error and then throw a new one with the 422 code attached and here you will just have one return without any ifs